specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-01-01/frontdoor.json

```diff
@@ -0,0 +1,2292 @@
+            "name": "rulesEngineParameters",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RulesEngine"
+            }
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/RulesEngine"
+            }
+          },
+          "201": {
+            "description": "Created. The request has been fulfilled and a new RulesEngine has been created.",
+            "schema": {
+              "$ref": "#/definitions/RulesEngine"
+            }
+          },
+          "202": {
+            "description": "Accepted. The request has been accepted for processing and the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/RulesEngine"
+            }
+          },
+          "default": {
+            "description": "Front Door error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create or update a specific Rules Engine Configuration": {
+            "$ref": "./examples/FrontdoorRulesEngineCreate.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "FrontDoors"
+        ],
+        "description": "Deletes an existing Rules Engine Configuration with the specified parameters.",
+        "operationId": "RulesEngines_Delete",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/frontDoorNameParameter"
+          },
+          {
+            "$ref": "#/parameters/rulesEngineNameParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. The request has been accepted for processing and the operation will complete asynchronously."
+          },
+          "204": {
+            "description": "No Content. The request has been accepted but the Rules Engine Configuration was not found."
+          },
+          "default": {
+            "description": "Front Door error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete Rules Engine Configuration": {
+            "$ref": "./examples/FrontdoorRulesEngineDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
>"x-ms- [](start = 8, length = 6)

For a Post LRO operation, add "x-ms-long-running-operation-options" extension with "final-state-via" property.
``` json
"x-ms-long-running-operation-options": {
  "final-state-via": "location"
}
```

or

``` json
"x-ms-long-running-operation-options": {
  "final-state-via": "azure-async-operation"
}```